google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -45,6 +54,11 @@
   private JTextPane warningMessage;
   private JPanel cloudSdkPanel;
   private JLabel warningIcon;
+  private JRadioButton managedRadioButton;
+  private JRadioButton customRadioButton;
+  private JCheckBox enableAutomaticUpdatesCheckbox;
```

Comment:
what about explicitly setting its default check/uncheck value and having a member boolean here containing that default? that way its easy to see what the default is and to change it later if we wanted.

Reply:
Sure, good idea.